CONTRIBUTORS.md

```diff
@@ -10,3 +10,15 @@ Please either submit a pull request adding your name, or mention it to another o
 and we will add you. **All** contributors belong here. 💯
 
 * [Bill Mulligan](https://github.com/xmulligan)
+* [Daniel Iziourov](https://github.com/danmx)
+* [Daniel Teixeira](https://github.com/daniel-f3)
+* Jietao Xiao (jietaoXiao@stu.xidian.edu.cn)
+* Jinku Li (jkli@xidian.edu.cn)
+* [Micah Hausler](https://github.com/micahhausler)
+* Nanzi Yang (nzyang@stu.xidian.edu.cn)
```

Comment:
@younaman can you confirm whether it is OK to publish you and your colleagues' emails like this?

Reply:
The alternative is that I can link your name here to your github profile, and just keep the names without emails for the others you are working with. I don't mind which way to present these, let us know what you think?

Reply:
It's OK, thanks! I just want my name and e-mail like this.

Reply:
I am not familiar with the Github's "approve changes", and I am sorry for the delay:) What you added is what I want, thank you again for your patience and work @joestringer.
Regards,
Nanzi Yang.

Reply:
It's no trouble, thanks!